pilot_urjc_bringup/params/pilot_modes.yaml

```diff
@@ -6,13 +6,19 @@ pilot:
     parts:
```

Comment:
I think all nodes and sub-systems defined in the file should be parts of the pilot, as long as they are needed in one of the modes.
Maybe @norro or @ralph-lange can clarify

Reply:
There is no need to have a single root in the system hierarchy, but the hierarchy may be formed by multiple trees. This may be beneficial to avoid a combinatorial explosion by the modes of unrelated (orthogonal) subsystems.
@norro, am I right that the multiple trees may be specified in one file together or is a separate file required for each tree?

Reply:
I understand your point, thanks @ralph-lange 
My question came from the fact that only the mode `LASER_ERROR` defines a mode for the `amcl`, but the other modes don't.
Is this interpreted by the mode manager as: those mode do not need the `acml` part?
Because if that is the case, then I think we should add a mode for the `amcl` in all the modes for the system `pilot`

Reply:
Sorry, I did not see this point. You're right, a state+mode should be defined for `amcl` in all modes of `pilot`. I assume that otherwise, the last state+mode will be kept, but @norro will know that immediately from his head.

Reply:
> Sorry, I did not see this point. You're right, a state+mode should be defined for `amcl` in all modes of `pilot`. I assume that otherwise, the last state+mode will be kept, but @norro will know that immediately from his head.

This is something to take into account for your ssytem_modes model @jginesclavero

Reply:
Of course, I will take into account and I will do another PR fixing it. Thank you so much for the revisions @ralph-lange @chcorbato